test/test_files/rdf/ddl.test

```diff
@@ -0,0 +1,14 @@
+-GROUP Rdf
+-DATASET CSV tinysnb
+
+--
+
+-CASE CreateRdfGraph
```

Comment:
Can you add more tests?
1. Execute create rdf inside an active manual transaction.
2. Test whether the table exists after the transaction is aborted.

Reply:
I'll leave a TODO here. It's not easy to add manual tax in test file yet.